web-ui/src/pages/PulseReportPage.jsx

```diff
@@ -51,6 +52,18 @@ const ociDarkBlue = '#2c519e';
 // const ociOrange = '#f8b576'; // too light
 const orange = '#b26801';
 
+const ScoreOption = {
```

Comment:
When viewing the 5 highest and lowest average scores, the user can choose between viewing only internal scores, only external scores, or a combination of internal and external scores.